gcs-action/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "gcs-action",
+  "version": "0.0.1",
+  "private": true,
+  "description": "GCS action",
+  "main": "lib/main.js",
+  "scripts": {
+    "build": "tsc",
+    "format": "prettier --write **/*.ts",
+    "format-check": "prettier --check **/*.ts",
+    "lint": "eslint src/**/*.ts",
+    "pack": "ncc build",
+    "test": "jest",
+    "all": "npm run build && npm run format && npm run lint && npm run pack && npm test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleCloudPlatform/github-actions/setup-gcloud.git"
+  },
+  "keywords": [
+    "actions",
+    "gcs",
+    "setup"
+  ],
+  "author": "GoogleCloudPlatform",
+  "license": "Apache 2.0",
+  "dependencies": {
```

Comment:
Please match all dependencies and dev dependencies with https://github.com/GoogleCloudPlatform/github-actions/pull/60

Reply:
Done - using @google-cloud/storage instead of auth. Also added Sinon as a devDep as I am using it for spies in test.